x-pack/plugins/enterprise_search/server/__mocks__/router.mock.ts

```diff
@@ -84,6 +80,16 @@ export class MockRouter {
   public shouldThrow = (request: MockRouterRequest) => {
     expect(() => this.validateRoute(request)).toThrow();
   };
+
+  findRouteRegistration = () => {
```

Comment:
💯 naming!

[random aside] I'm very much a yoloscripter, but it's never super clear to me how much you _have_ to use `public` vs `private` in classes 🤔 like does `private findRouteRegistration` do anything here or is it basically just a developer hint?

Reply:
Oh gosh, I missed making this private. No, I think they do have functional significance. It looks like they're experimental though: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#field_declarations. I'll update this to be private.

Reply:
b43820e333e78bb98babf8c0f5d6eea832e3c207